sdks/python/test-suites/tox/py38/build.gradle

```
@@ -113,6 +113,18 @@ toxTask "testPy38pytorch-110", "py38-pytorch-110", "${posargs}"
 test.dependsOn "testPy38pytorch-110"
 preCommitPy38.dependsOn "testPy38pytorch-110"
 
+toxTask "testPy38pytorch-111", "py38-pytorch-111", "${posargs}"
+test.dependsOn "testPy38pytorch-111"
+preCommitPy38.dependsOn "testPy38pytorch-111"
+
+toxTask "testPy38pytorch-112", "py38-pytorch-112", "${posargs}"
+test.dependsOn "testPy38pytorch-112"
+preCommitPy38.dependsOn "testPy38pytorch-112"
+
+toxTask "testPy38pytorch-113", "py38-pytorch-113", "${posargs}"
+test.dependsOn "testPy38pytorch-113"
+preCommitPy38.dependsOn "testPy38pytorch-113"
```

```suggestion
preCommitPyCoverage.dependsOn "testPy38pytorch-111"

toxTask "testPy38pytorch-112", "py38-pytorch-112", "${posargs}"
test.dependsOn "testPy38pytorch-112"
preCommitPyCoverage.dependsOn "testPy38pytorch-112"

toxTask "testPy38pytorch-113", "py38-pytorch-113", "${posargs}"
test.dependsOn "testPy38pytorch-113"
preCommitPyCoverage.dependsOn "testPy38pytorch-113"
```

These were generally updated in https://github.com/apache/beam/commit/b3aa2e89489898f8c760294ba4dba2310ac53e70